rest/src/main/java/org/n52/io/extension/RenderingHintsExtension.java

```diff
@@ -74,7 +74,11 @@ public void addExtraMetadataFieldNames(DatasetOutput output) {
     }
 
     private boolean hasRenderingHints(DatasetOutput output) {
-        return hasSeriesConfiguration(output) || hasPhenomenonConfiguration(output);
+        return hasDatasetParameters(output) && (hasSeriesConfiguration(output) || hasPhenomenonConfiguration(output));
+    }
+
+    private boolean hasDatasetParameters(DatasetOutput output) {
+        return output.getDatasetParameters() != null;
```

Comment:
Took longer than expected .. but this should be a runnable/testable version:

* https://github.com/SpeckiJ/series-rest-api/pull/1
* https://github.com/SpeckiJ/dao-series-api/pull/2 

**Note:** Currently, the [DAO build fails on Travis](https://travis-ci.org/ridoo/dao-series-api/builds/267964450?utm_source=github_status&utm_medium=notification) as API and constants from series-rest-api have been changed slightly